jwst/regtest/test_niriss.py

```diff
@@ -2,68 +2,44 @@
 import os
 
 import pytest
-from astropy.io.fits.diff import FITSDiff
 
-from jwst.lib.suffix import replace_suffix
-from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
-from jwst.stpipe import Step
-
-
-def is_like_truth(rtdata, fitsdiff_default_kwargs, suffix, truth_path='truth/niriss/test_niriss'):
-    """Compare step outputs with truth"""
-
-    # If the input was an association, the output should be the name of the product
-    # Otherwise, output is based on input.
-    if rtdata.asn:
-        output = rtdata.asn['products'][0]['name']
-    else:
-        output = os.path.splitext(os.path.basename(rtdata.input))[0]
-    output = replace_suffix(output, suffix) + '.fits'
-    rtdata.output = output
-
-    rtdata.get_truth(os.path.join(truth_path, output))
-
-    diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
-    assert diff.identical, diff.report()
+from . import regtestdata as rt
 
 
 @pytest.fixture(scope='module')
-def run_nrs_wfss_spectral(jail, rtdata_module):
+def run_nis_wfss_spectral(jail, rtdata_module):
```

Comment:
Wouldn't hurt to just spell out the whole instrument name (i.e. "niriss"), which also avoids all potential for confusion.